### ISSUE TITLE
[Doc]: Accessibility issues on status tags

### ISSUE THREAD

**Issue by CyriaqueBillard** [Orange-OpenSource/ouds-flutter#774]
### Documentation Location

https://github.com/Orange-OpenSource/ouds-flutter#status

### Current issue

In some tags in the status section, the combination of the foreground colour with background colour does not meet the AA contrast ratio.

### Expected Correction

**Orange tag**
Change typo colour to #000000

**Yellow tag**
Change typo colour to #000000

**Light green tag**
Change typo colour to #000000

**Dark green tag**
Change typo colour to #000000

**Blue tag**
Change typo colour to #000000

**Light grey tag**
Change typo colour to #000000
